ultraScurve.py

```diff
@@ -1,7 +1,10 @@
-#!/bin/env python
+#!/bin/env python2.7
 """
 Script to take Scurve data using OH ultra scans
 By: Cameron Bravo (c.bravo@cern.ch)
+
+Modified By: 
+	Brian Dorney (brian.l.dorney@cern.ch)
```

Comment:
Looks like you modified how mask is set. If this is enough to add your name here, add study too.